server/utils/tasks/listTable.ts

```diff
@@ -28,6 +39,49 @@ const getTaskType = (task: Task): string => {
   return taskType
 }
 
+const assessmentDecisionTranslations: Record<AssessmentDecision, string> = {
+  accepted: 'Accepted',
+  rejected: 'Rejected',
+}
+
+const placementApplicationDecisionTranslations: Record<PlacementApplicationDecision, string> = {
+  accepted: 'Accepted',
+  rejected: 'Rejected',
+  withdraw: 'Withdraw',
+  withdrawn_by_pp: 'Withdrawn by pp',
+}
+
+const placementRequestTaskOutcomeTranslations: Record<PlacementRequestTaskOutcome, string> = {
+  matched: 'Matched',
+  unable_to_match: 'Unable to match',
+}
+
+const getAssessmentDecisionOutcome = (outcome: AssessmentDecision): string => {
+  return assessmentDecisionTranslations[outcome]
+}
+
+const getPlacementApplicationDecisionOutcome = (outcome: PlacementApplicationDecision): string => {
+  return placementApplicationDecisionTranslations[outcome]
+}
+
+const getPlacementRequestTaskOutcome = (outcome: PlacementRequestTaskOutcome): string => {
+  return placementRequestTaskOutcomeTranslations[outcome]
+}
+
+export const getDecisionOutcome = (task: Task): string => {
```

Comment:
In the After screenshot it looks like this function is returning `undefined` 
![image](https://github.com/ministryofjustice/hmpps-approved-premises-ui/assets/44123869/601531ec-f1d5-487a-9819-67b2d459aa09)